cove_ocds/test_input.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
yep, I tried to come up with a solution for not doing this but failed when I was rushing to get this done.  I really needed class based testing but it acted weirdly with fixtures and did not want to spend ages figuring it out.  Definitely something to fix for the future